db/seeds/dummy_structures.rb

```diff
@@ -19,12 +19,25 @@
   Partnership.create!(school: School.first, lead_provider: LeadProvider.first)
 end
 
-unless AdminProfile.first || Rails.env.production?
-  user = User.find_or_create_by!(email: "ecf@mailinator.com") do |u|
+if Rails.env.deployed_development? && User.none?
+  user = User.find_or_create_by!(email: "admin@example.com") do |u|
     u.full_name = "Admin User"
+    u.confirmed_at = Time.zone.now.utc
   end
-  user.confirm
   AdminProfile.create!(user: user)
+
+  user = User.find_or_create_by!(email: "lead-provider@example.com") do |u|
+    u.full_name = "Lp User"
+    u.confirmed_at = Time.zone.now.utc
+  end
+  user.confirm
```

Comment:
That can be canned now I think

Reply:
Can still be canned.

Reply:
Indeed! Even if it weren't for this change, it would still be unnecessary as all it does is set confirmed_at